cfme/tests/cloud_infra_common/test_tag_objects.py

```diff
@@ -14,14 +14,12 @@
 from cfme.infrastructure.provider import InfraProvider
 from cfme.infrastructure.virtual_machines import Vm, Template
 from cfme.web_ui import Quadicon, mixins, toolbar as tb
-from utils import providers
 from utils.appliance.implementations.ui import navigate_to
 
 
 @pytest.fixture(scope="module")
-def setup_first_provider():
-    providers.setup_a_provider_by_class(InfraProvider)
-    providers.setup_a_provider_by_class(CloudProvider)
+def cloud_and_infra_provider(cloud_provider, infra_provider):
```

Comment:
Should I be solving this here though? I mean if we have a problem with RAM and we can't have 2 providers on an appliance at the same time, that's an issue that needs solving for all tests not just here.